pkg/custom_detectors/custom_detectors.go

```diff
@@ -166,14 +168,30 @@ func (c *CustomRegexWebhook) createResults(ctx context.Context, match map[string
 			}
 			req.Header.Add(key, strings.TrimLeft(value, "\t\n\v\f\r "))
 		}
-		res, err := httpClient.Do(req)
+		resp, err := httpClient.Do(req)
 		if err != nil {
 			continue
 		}
-		// TODO: Read response body.
-		res.Body.Close()
-		if res.StatusCode == http.StatusOK {
+		defer func() {
+			_, _ = io.Copy(io.Discard, resp.Body)
+			_ = resp.Body.Close()
+		}()
+
+		if resp.StatusCode == http.StatusOK {
+			// mark the result as verified
 			result.Verified = true
+
+			body, err := io.ReadAll(resp.Body)
+			if err != nil {
+				continue
+			}
+
+			// TODO: handle different content-type responses seperatly when implement custom detector configurations
+			responseStr := string(body)
+
+			// store the processed response in ExtraData
+			result.ExtraData["response"] = responseStr
```

Comment:
@kashifkhan0771 **Thought**: storing the entire response could lead to some large outputs. This could be annoying for users who already are using custom detectors. 

One option could be limit the response to 200 characters. I don't feel comfortable merging this PR as is. 

@kashifkhan0771 additionally, I tried spinning up a verification server and testing this myself but found that the only thing that could printed in the output was the name of the customer verifier: `"ExtraData":{"name":"HogTokenDetector"}` when using the `--json` flag. A mock test might help here, although I have a suspicion there might be some other issues considering I'm not able to surface additional `ExtraData` past the name of the custom verifier in the output.

@CameronLonsdale what is your use case for `ExtraData`? How are you using TruffleHog to surface this data (what command and flags)?

Reply:
For me, I always run trufflehog with --json --only-verified. I wanted to surface data from my custom patterns that are the responses from verifying the credential against the API. So this would typically be a JSON blob which tells me info about the user which corresponds to the token I just verified.

I wrap trufflehog around custom code to sync results to Jira etc so hence why JSON. But I would expect the extraData to show up in CLI output aswell, so truncating it to be readable at the terminal makes sense.

Reply:
@zricethezav @CameronLonsdale I added the limit of 200 chars for response. I am not sure how to test the custom detector. If anyone of you can guide me how to do it. I can test locally to understand why it is not printing the extra data in output.

Reply:
Thank you, @mcastorina, for your guidance on running the custom detector. I successfully tested it and identified the issue that was preventing the results from printing. I've fixed it now. Please review my changes.